lib/topological_inventory/ansible_tower/operations/source.rb

```diff
@@ -25,6 +25,7 @@ def availability_check
             return
           end
 
+          logger.info("Checking availability of source #{source_id}")
```

Comment:
also duplicate of processor.rb log message...
we should move all these common things to common gem